ppr-ui/src/components/mhrRegistration/HomeLocation/HomeCivicAddress.vue

```diff
@@ -0,0 +1,135 @@
+              <v-textarea
+                autocomplete="new-password"
+                auto-grow
+                filled
+                class="street-address-additional"
+                label="Additional Street Address (Optional)"
+                :name="Math.random()"
+                rows="1"
+                v-model="address.streetAdditional"
+                :rules="[...addressSchema.streetAdditional]"
+              />
+            </div>
+            <div class="form__row two-column">
+              <v-row>
+                <v-col>
+                  <v-text-field
+                    autocomplete="new-password"
+                    filled
+                    class="item address-city"
+                    label="City"
+                    :name="Math.random()"
+                    v-model="address.city"
+                    :rules="[...addressSchema.city]"
+                  />
+                </v-col>
+                <v-col>
+                  <v-text-field
+                    filled
+                    :disabled=true
+                    class="item address-region"
+                    label="Province"
+                    hint="Address must be in B.C."
+                    :name="Math.random()"
+                    v-model="address.region"
+                    :rules="[...addressSchema.region]"
+                  />
+                </v-col>
+              </v-row>
+            </div>
+          </v-form>
+      </div>
+      </v-col>
+    </v-row>
+  </v-card>
+</template>
+
+<script lang="ts">
+/* eslint-disable no-unused-vars */
+import { defineComponent, reactive, toRefs, watch } from '@vue/composition-api'
+import { AutoComplete } from '@/components/search'
+import { BaseAddress } from '@/composables/address'
+import { AddressIF } from '@/composables/address/interfaces'
+import { PartyAddressSchema } from '@/schemas/party-address'
+
+/* eslint-enable no-unused-vars */
+
+export default defineComponent({
+  name: 'HomeCivicAddress',
+  components: {
+    BaseAddress,
+    AutoComplete
+  },
+  props: {
+    /* used for readonly mode vs edit mode */
+    editing: {
+      type: Boolean,
+      default: false
+    }
+  },
+  setup (props, context) {
+    /* const {} = useActions<any>([]) */
+    const initAddress : AddressIF = {
+      street: '',
+      streetAdditional: '',
+      city: '',
+      region: 'British Columbia',
+      country: '',
+      postalCode: ''
+    }
+
+    const address : AddressIF = {
+      street: initAddress.street,
+      city: initAddress.city,
+      region: initAddress.region,
+      postalCode: initAddress.postalCode,
+      country: initAddress.country
+    }
+
+    const addressSchema = PartyAddressSchema
+
+    const localState = reactive({
+      isValidLot: false,
+      showAllAddressErrors: false
+    })
+    localState.isValidLot = true
+    /** Apply local models to store when they change. **/
+    /** Clear/reset forms when select option changes. **/
```

Comment:
It might make sense to initialize your `address` into localstate, which you can then tie the inputs too using the v-model. 

After which you will need to watch for changes and apply to store by writing an Action/Mutation for this components data. 

Also looks like these properties are perhaps unrelated to this component.